finality-aleph/src/sync/substrate/mod.rs

```diff
@@ -137,12 +137,12 @@ impl<H: SubstrateHeader<Number = BlockNumber>> JustificationT for Justification<
 }
 
 /// Translates raw aleph justifications into ones acceptable to sync.
-pub trait JustificationTranslator<H: SubstrateHeader<Number = BlockNumber>>: Send + Sync {
+pub trait JustificationTranslator: Send + Sync {
```

Comment:
The fact that this is no longer parametrized kinda suggests this shouldn't be a trait. Perhaps instead move the function straight to `SubstrateChainStatus` and operate on that directly everywhere where `JustificationTranslator` has been used? This might turn out to be a much bigger change though, so perhaps not for this PR, mentioning it since it would solve one of the other comments by accident, plus it's worth to consider in general imo.

Reply:
now it's a struct